ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json

```diff
@@ -83,8 +83,8 @@
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorAppReference", "Label": "Your reference for this application", "FieldType": "Text", "SecurityClassification": "Public", "Max": 100},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorIsExec", "Label": "Is the probate practitioner named in the will as an executor?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorWillSignSOT", "Label": "Are you the probate practitioner who will sign the statement of truth?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorIsApplying", "Label": "Is the probate practitioner applying?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorNotApplyingReason", "Label": "Why aren't they applying?", "FieldType": "FixedRadioList", "FieldTypeParameter": "solsNotApplyingReasonFixedList", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorIsApplying", "Label": "Is the probate practitioner acting as an executor?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorNotApplyingReason", "Label": "Why are they not applying?", "FieldType": "FixedRadioList", "FieldTypeParameter": "solsNotApplyingReasonFixedList", "SecurityClassification": "Public"},
```

Comment:
I'm pretty sure we changed this from are not to aren't; James may have been working from out of date screenshots - best to check with Narm

Reply:
Switched back to aren't after speaking with Narm